Add note about dialyzer high CPU usage to readme and issue template. (#293)
elixir-lsp/elixir-ls: .github/ISSUE_TEMPLATE.md

- [ ] Stop your editor, remove the entire `.elixir_ls` directory, then restart your editor
  * NOTE: This will cause you to have to re-run the dialyzer build for your project

> If you're experiencing high CPU usage, it is most likely Dialyzer building the PLTs; after it's done the CPU usage should go back to normal You could also disable Dialyzer in the settings.

Author: ```suggestion
If you're experiencing high CPU usage, it is most likely Dialyzer building the PLTs; after it's done the CPU usage should go back to normal. You could also disable Dialyzer in the settings.
```

> If you're experiencing high CPU usage, it is most likely Dialyzer building the PLTs, after it builds those the CPU usage should go back to normal, but you could also disable Dialyzer in the settings.

Reviewer: ```suggestion
If you're experiencing high CPU usage, it is most likely Dialyzer building the PLTs; after it's done the CPU usage should go back to normal You could also disable Dialyzer in the settings.
```
